ddp-workspace/projects/ddp-testboston/src/app/components/header/header.component.html

```diff
@@ -59,7 +60,7 @@
     </div>
 </header>
 
-<div #overlay class="overlay">
+<div *ngIf="!isAdmin" #overlay class="overlay">
```

Comment:
Found one issue - we also need to hide overlay panel for admins. Otherwise, it appeared in the DOM but was hidden by display: none.